pkg/preference/preference_test.go

```diff
@@ -175,7 +175,7 @@ func TestSetConfiguration(t *testing.T) {
 		t.Fatal(err)
 	}
 	defer tempConfigFile.Close()
```

Comment:
maybe add a test case for experimental mode here

Reply:
Done